src/ref-field/ref-field.component.html

```diff
@@ -8,7 +8,8 @@
       </div>
     </div>
     <div *ngIf="!isTemplateEnabled">
-      <a target="_blank" class="break-word" [href]="anchorHref">{{anchorDisplay}}</a>
+      <string-input *ngIf="shouldDisplayInputField" [pathString]="pathString" [value]="ref" [tabIndex]="tabIndex" [placeholder]="schema.title"></string-input>
```

Comment:
I think this shouldn't be under `*ngIf="!isTemplateEnabled"`, we might want to display template (currently not used) even if `displayInputField` is enabled.

Reply:
`pathString` for this should be `<pathString>/$ref`

Reply:
Depending on the schema and what makes sense, maybe it's better to use `schema.properties.$ref.title` instead as placeholder.

Reply:
`[value]` should be actual `$ref` value, not hardcoded string.